Update agreements.md

mollyamcintyre opened the pull request:
updated agreement status section to re-format and revise to note office hours availability

mollyamcintyre committed:
Update agreements.md

updated agreement status section to re-format and revise to note office hours availability

lsgitter reviewed (changes requested):
Line 57: [Master Agreements Pipeline] should be [TTS Agreements Report]

mollyamcintyre committed:
Update agreements.md

revised "master agreements pipeline" to "tts agreements report" with minor edits
